src/test/njs_unit_test.c

```diff
@@ -16281,6 +16281,12 @@ static njs_unit_test_t  njs_test[] =
     { njs_str("Date.parse('2011-06-24T06:01:02.6255555Z')"),
       njs_str("1308895262625") },
 
+    { njs_str("Date.parse('2011-06-24T06:01:02.625555555Z')"),
+      njs_str("1308895262625") },
+
+    { njs_str("Date.parse('2011-06-24T06:01:02.6255555555Z')"),
+      njs_str("NaN") },
```

Comment:
Chrome, node and FireFox accepts 3 digits, and just skip all trailing digits in ms field. (in contrast qjs rounds last digit in case if there is more than 3 digits,). Why not behave here like de-facto chrome and friends?